org.mwc.debrief.core/src/org/mwc/debrief/core/wizards/s2r/TMAFromSensorWizard.java

```diff
@@ -78,7 +78,7 @@ public void addPages()
     }
 
     enterSolutionPage = new EnterSolutionPage(null, PAGE_TITLE,
-        "This page lets you enter an initial solution", imagePath, helpContext,
+        "Specify the Course and Speed initial values", imagePath, helpContext,

```

Comment:
Slight re-wording, for better English, please:
 "Specify the **initial** Course and Speed values"

Reply:
oh sure